criu/cr-dump.c

```diff
@@ -846,8 +847,10 @@ static int dump_task_thread(struct parasite_ctl *parasite_ctl,
 				const struct pstree_item *item, int id)
 {
 	struct parasite_thread_ctl *tctl = dmpi(item)->thread_ctls[id];
+	struct parasite_dump_cgroup_args cgroup_args;
 	struct pid *tid = &item->threads[id];
 	CoreEntry *core = item->core[id];
+	ThreadCoreEntry *thread_core = core->thread_core;
```

Comment:
since you only use `thread_core` in one please, I'd remove the variable and user `core->thread_core` directly.